catalogs/catalog-lakehouse-paimon/src/main/java/com/datastrato/gravitino/catalog/lakehouse/paimon/PaimonCatalogMetastore.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+package com.datastrato.gravitino.catalog.lakehouse.paimon;
+
+/** The type of Paimon catalog metastore. */
+public enum PaimonCatalogMetastore {
+  FILE,
```

Comment:
FILE or HADOOP?

Reply:
@qqqttt123, FILE.

Reply:
This catalog relies on the rename operation of HDFS? Does Paimon  the concept of `FILE`? I remember that Iceberg uses `HadoopCatalog` for similar concepts.

Reply:
@qqqttt123, I rename to `FILESYSTEM` which refers to [creating-catalogs](https://paimon.apache.org/docs/master/how-to/creating-catalogs/).

Reply:
OK for me. We should keep the consistent concepts with Paimon. It will reduce the cost of understanding.